templates/graphdb-provision-user-secret.yaml

```diff
@@ -0,0 +1,16 @@
+# Secret used from the jobs to authenticate to running GraphDB instances
+apiVersion: {{ .Values.versions.secret }}
+kind: Secret
+metadata:
+  name: graphdb-provision-user
+  labels:
+    {{- include "graphdb.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade, pre-rollback, post-install, post-upgrade, post-rollback
+    "helm.sh/hook-delete-policy": before-hook-creation, hook-succeeded, hook-failed
+    "helm.sh/hook-weight": "-9"
+type: Opaque
+data:
+  provisioningUsername: {{ .Values.graphdb.security.provisioningUsername | b64enc | quote }}
```

Comment:
do we have to encrypt the username?

Reply:
Yes, all the data in the secret is expected to be. I don't think you can even create the secret if the data is not b64enc